docs/evm/ethereum-compatibility/precompiles/pink.md

```diff
@@ -0,0 +1,9 @@
+# PINK Precompile
+
+TODO: Add pink token introduction
```

Comment:
@AurevoirXavier Ask for your help to add the PINK token overview here. I don't know much about this token. the USDT page FYI.